src/main/java/first/dto/StudDto.java

```diff
@@ -0,0 +1,112 @@
+package first.dto;
+
+public interface StudDto {
+
+    String FORM = "<html>\n" +
```

Comment:
FORM is so common name, could you add some word to this name? For example: STUDENT_PROPERTIES_FORM